src/ir_Magiquest.cpp

```diff
@@ -0,0 +1,105 @@
+#include "ir_Magiquest.h"
+#include <algorithm>
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRtimer.h"
+#include "IRutils.h"
+
+// #define DEBUG
+
+#define IS_ZERO(m, s) (((m) * 100 / ((m) + (s))) <= MAGIQUEST_ZERO_RATIO)
+#define IS_ONE(m, s)  (((m) * 100 / ((m) + (s))) >= MAGIQUEST_ONE_RATIO)
+
+// Strips taken from:
+// https://github.com/kitlaan/Arduino-IRremote/blob/master/ir_Magiquest.cpp
+// and
+// https://github.com/mpflaga/Arduino-IRremote
+
+// Source: https://github.com/mpflaga/Arduino-IRremote
+
+#if SEND_MAGIQUEST
+void IRsend::sendMagiQuest(uint32_t wand_id, uint16_t magnitude)
```

Comment:
Please use the standard arguments (as per other protocols) for this procedure.
i.e. data, nbits, repeat etc.

To convert from wand_id and magnitude etc, make a `encodeMagiQuest()` procedure that produces the correct "data" for the `sendMagiQuest()` method. e.g. Like done in NEC.